### ISSUE TITLE
Kobo Books DeDRM Failure

### ISSUE THREAD

**Issue by joeywen0830** [noDRM/DeDRM_tools#537]
### Question / bug report

I have multiple kobo ebooks, since kobo website no longer provides downloading, I tried to use dedrm instead, but none of my books can successful dedrm. Changing computers or callibre versions didn't help solve the issue.

### Which version of Calibre are you running?

7.8

### Which version of the DeDRM plugin are you running?

v10.0.9

### Log output

```log
Paste log output here.
```

**Comment by ElleKayEm** [comment]
You're trying to use Obok with Kobo Desktop or a Kobo ereader?  Are they purchased books or Kobo Plus?  Which Kobo website no longer has downloads?  (I ask because the US site still does.)

**Comment by joeywen0830** [comment]
> You're trying to use Obok with Kobo Desktop or a Kobo ereader? Are they purchased books or Kobo Plus? Which Kobo website no longer has downloads? (I ask because the US site still does.)

With Kobo desktop. They are purchased books not from kobo plus. I don't see download option on TW JP US versions of the website, or do you have one that can currently download acsm?

**Comment by ElleKayEm** [comment]
I can still download the ACSM files (or EPUB if DRM-free) when I'm logged into Kobo's US site.  And I am still able to use Obok with Kobo Desktop.  For a while now, I've heard of Kobo Plus books not working with Obok and not having an option for downloading ACSM files (Adobe DRM).  I wonder if Kobo is changing so purchased books are the same.

**Comment by joeywen0830** [comment]
> I can still download the ACSM files (or EPUB if DRM-free) when I'm logged into Kobo's US site. And I am still able to use Obok with Kobo Desktop. For a while now, I've heard of Kobo Plus books not working with Obok and not having an option for downloading ACSM files (Adobe DRM). I wonder if Kobo is changing so purchased books are the same.

Did you used both of these methods recently?

**Comment by ElleKayEm** [comment]
Yes, just checked and both are still working for me as of today.

**Comment by HamsterExAstris** [comment]
[Kobo's help docs](https://help.kobo.com/hc/en-us/articles/360019527954-Download-books-from-your-Kobo-account-to-export-to-another-device-or-app) specifically lists Japan as the only impacted country (for now).

**Comment by joeywen0830** [comment]
> [Kobo's help docs](https://help.kobo.com/hc/en-us/articles/360019527954-Download-books-from-your-Kobo-account-to-export-to-another-device-or-app) specifically lists Japan as the only country unable to download through ADE (for now).

Ah, thanks for the info. My books are indeed Japanese, perhaps that's why I can't download them.
